docs/guides/unbound.md

```diff
@@ -52,82 +52,154 @@ The first thing you need to do is to install the recursive DNS resolver:
 sudo apt install unbound
 ```
 
-**Important**: Download the current root hints file (the list of primary root servers which are serving the domain "." - the root domain). Update it roughly every six months. Note that this file changes infrequently.
```

Comment:
> Removed the steps to download the root.hints file, since they aren't necessary.

I disagree. We recommend users to download the current IP addresses of the root zone.
Your suggestion is, instead, to *hardcode* them in the config file (your proposed lines 171-186). Why do you think this is a better way to go?

Reply:
The [Unbound](https://packages.debian.org/buster/unbound) Debian package installed via apt, comes with [unbound-anchor](https://packages.debian.org/buster/unbound-anchor) as a dependency, and the root.hints file is downloaded automatically. The systemd service checks for the root.hints at `ExecStartPre=-/usr/lib/unbound/package-helper root_trust_anchor_update`.